tests/foreman/api/test_repository.py

```diff
@@ -388,6 +388,26 @@ def test_positive_create_checksum(self):
                 ).create()
                 self.assertEqual(checksum_type, repo.checksum_type)
 
+    @tier1
+    @run_only_on('sat')
+    def test_positive_create_checksum_with_background_policy(self):
+        """Attempt to create repository with checksum and background policy.
+
+        :id: 4757ae21-f792-4886-a86a-799949ad82d0
+
+        :expectedresults: A repository is created and has expected checksum type.
+
+        :CaseImportance: Critical
+        """
+        for checksum_type in 'sha1', 'sha256':
```

Comment:
Use constants from https://github.com/SatelliteQE/robottelo/blob/master/robottelo/constants.py#L224

Reply:
ahh , we could use that but as per my understanding does not make much difference as naming is still using same in constants  for key.

it's pushed now , good to merge.